src/routes/rasa.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Switch, Route, useRouteMatch } from 'react-router-dom'
+import queryString from 'query-string'
+
+import RasaPage from '../pages/rasa'
+
+const USER_ID_QUERY_PARAMETER = 'uid'
+const RASA_GREET_INTENT = 'greet'
+const RASA_CHECKIN_INTENT = 'daily_checkin'
+const RASA_METADATA_ENTITY = 'metadata'
+const RASA_TIMEZONE_PARAMETER = 'timezone'
+const RASA_USER_ID_PARAMETER = 'user_id'
+
+function parseQueryString() {
+  const hash = window.location.hash
+  return queryString.parse(hash.substr(hash.indexOf('?')))
```

Comment:
Message to the reviewer: is there a reason why we're using the `HashRouter` instead of `BrowserRouter`? Using the `BrowserRouter` would allow better handling of the query-string.

Reply:
Hosted on s3 so if browser history is enabled refresh will result in 404